src/main/java/seedu/programmer/logic/commands/AddCommand.java

```diff
@@ -30,8 +30,13 @@
             + PREFIX_EMAIL + "e0518441@u.nus.edu";
 
     public static final String MESSAGE_SUCCESS = "New student added: %1$s";
+    public static final String MESSAGE_DUPLICATE_STUDENT_ID = "This student with the same Student Id "
+            + "already exists in the ProgrammerError";
+    public static final String MESSAGE_DUPLICATE_STUDENT_EMAIL = "This student with the same Email "
```

Comment:
Should 'Email' be lower case here? The user doesn't need to know that Email is represented as a class in our code.

Reply:
sounds good!